src/main/java/com/codereligion/bugsnag/logback/Converter.java

```diff
@@ -71,6 +73,17 @@ public NotificationVO convertToNotification(final ILoggingEvent loggingEvent) {
         return Collections.singletonList(event);
     }
 
+    private String convertToSeverity(final Level level) {
+        if(level==Level.ERROR){
+            return "error";
+        }else if(level == Level.WARN){
+            return "warning";
+        }else if(level == Level.INFO){
+            return "info";
+        }
+        return null;
```

Comment:
OK, but I think we should update the README.md and make it clear that the logging levels: OFF, DEBUG, TRACE and ALL are currently not handled and will be sent to bugsnag as `null`